Sources/PubNubChatComponents/Views/Reaction/Provider/ReactionProvider.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ReactionProvider.swift
+//  PubNubChat
+//
+//  Created by Maciej Adamczyk on 25/08/2022.
+//
+
+import Foundation
+
+public protocol ReactionProvider {
```

Comment:
Do we need any Swift comments above this declaration? What's the purpose of this protocol, what strings are allowed to pass, etc.